cava: update to 0.8.2.

skmpz opened the pull request:
<!-- Uncomment relevant sections and delete options which are not applicable -->

#### Testing the changes
- I tested the changes in this PR: **YES**

<!--
#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**|**NO**
-->

<!-- Note: If the build is likely to take more than 2 hours, please add ci skip tag as described in
https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#continuous-integration
and test at least one native build and, if supported, at least one cross build.
Ignore this section if this PR is not skipping CI.
-->
<!-- 
#### Local build testing
- I built this PR locally for my native architecture, (ARCH-LIBC)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl
  - armv7l
  - armv6l-musl
-->

Piraty commented:
0.8.2 is out, note [c1dc828499](https://github.com/karlstav/cava/commit/c1dc8284990bb5f0f3cebb171fbd33a8e9813e01)

skmpz commented:
> 0.8.2 is out, note [c1dc828499](https://github.com/karlstav/cava/commit/c1dc8284990bb5f0f3cebb171fbd33a8e9813e01)

updated to 0.8.2 and removed the xxd dependency.
